avalancheup-aws/src/install_subnet_chain/mod.rs

```diff
@@ -1012,7 +1022,33 @@ pub async fn execute(opts: Flags) -> io::Result<()> {
         }
     }
 
-    println!();
+    // If an alias for the chain is provided, set the alias here.
+    // Ref: https://docs.avax.network/apis/avalanchego/apis/admin#adminaliaschain
+    if !opts.chain_alias.is_empty() {
+        match json_client_admin::alias_chain(
+            &opts.chain_rpc_url,
+            opts.chain_name.clone(),
+            opts.chain_alias.clone(),
+        )
+        .await
+        {
+            Ok(_) => log::info!(
+                "set alias {} for chain {}",
+                &opts.chain_alias,
+                &opts.chain_name
+            ),
+            Err(e) => {
+                return Err(Error::new(
```

Comment:
I see, sure that makes sense. If the admin API is not enabled it's not possible to even set the alias right? In that case a warning should be emitted. We should also update the helptext for the command to explain that the admin API needs to be enabled via the avalanchego spec config to set a chain alias successfully.

Reply:
> If the admin API is not enabled it's not possible to even set the alias right

You are right.

> update the helptext for the command to explain that the admin

+1.

Since this alias API call is not critical for setting up the network, just warning + help text will suffice.